### ISSUE TITLE
Repr truncated for objects

### ISSUE THREAD

**Issue by laike9m** [alexmojaki/cheap_repr#15]
This is another case where a not so long repr is truncated. 

```python
from cheap_repr import cheap_repr

def this_is_my_function():
    class A:
        pass

    a = A()

    print(cheap_repr(a))


this_is_my_function()
```

gives `<__main__.this_is_my_functio...als>.A object at 0x1096c8040>`

**Comment by alexmojaki** [comment]
Fixed in https://github.com/alexmojaki/cheap_repr/commit/8c5ab161cb3944c9e2f048ade43471376b10165d, released in 0.4.4

**Comment by laike9m** [comment]
Thanks a lot!
